float32 altitude    		# altitude to take off the drone
---
bool ack 			# acknowledged information